docs/streams/developer-guide/datatypes.html

```diff
@@ -163,6 +164,72 @@ <h3>JSON<a class="headerlink" href="#json" title="Permalink to this headline"></
         <p>As shown in the example, you can use JSONSerdes inner classes <code class="docutils literal"><span class="pre">Serdes.serdeFrom(&lt;serializerInstance&gt;, &lt;deserializerInstance&gt;)</span></code> to construct JSON compatible serializers and deserializers.
         </p>
       </div>
+      <div class="section" id="window-serdes">
+        <h3>Window Serdes<a class="headerlink" href="#window-serdes" title="Permalink to this headline"></a></h3>
+        <p>Apache Kafka Streams includes serde implementations for windowed keys in
```

Comment:
```suggestion
        <p>Apache Kafka Streams includes serde implementations for windowed types in
```
While it's usually used for keys, there is not such entanglement, so using `types` is more generic.

Reply:
fixed in ad04ddb